### ISSUE TITLE
fix verbatim block with Emacs 27

### ISSUE THREAD

**Issue by mmitch** [mmitch/ox-bb#3]
After expanding the build pipeline to include Emacs 27, a bug popped up:

```
#!/bin/bash -eo pipefail
./run-tests.sh ox-bb.el

running testing/test-bold OK
running testing/test-code OK
running testing/test-entity OK
running testing/test-fixed-width FAILED
Org mode version 9.3 (release_9.3 @ /usr/local/share/emacs/27.1/lisp/org/)
--- testing/test-fixed-width.output	2021-01-06 22:11:28.609788870 +0000
+++ testing/test-fixed-width.bbcode	2021-01-06 22:11:33.105776887 +0000
@@ -2,7 +2,7 @@ paragraph 1
 
 [code]
 verbatim line
-  indented verbatim line
-[/code]
+  indented verbatim line[/code]
+
 
 paragraph 2

running testing/test-footnote-multiple OK
running testing/test-footnote-plain OK
running testing/test-geshi-block-without-language OK
running testing/test-geshi-block OK
running testing/test-headline-lv1 OK
running testing/test-headline-lv2 OK
running testing/test-headline-lv3 OK
running testing/test-headline-lv4 OK
running testing/test-headline-lv5 OK
running testing/test-italic OK
running testing/test-line-break OK
running testing/test-link-about OK
running testing/test-link-empty OK
running testing/test-link-encode-url-only-once OK
running testing/test-link-encode-url OK
running testing/test-link-http OK
running testing/test-link-https OK
running testing/test-multiline-paragraph OK
running testing/test-multiple-paragraphs OK
running testing/test-plain-list-descriptive OK
running testing/test-plain-list-ordered OK
running testing/test-plain-list-unordered OK
running testing/test-quote-block OK
running testing/test-single-paragraph OK
running testing/test-strike-through OK
running testing/test-underline OK
running testing/test-verbatim OK
1 of 31 tests failed.


Exited with code exit status 1

CircleCI received exit code 1
```

All other tests work, so this seems like an incompatible change in Org itself as newer Emacen include newer version of Org.

Basically the final newline in a verbatim block is now outside the block instead of inside it.

**Comment by mmitch** [comment]
It is actually a regression introduced with Org 9.1.14, more exactly 7d9e4da447fa54633daf75a241c6b0a603c0e662.
Other exporters show the same problem.
I will post on the orgmode-ml about this.

**Comment by mmitch** [comment]
fixed with c6dd3ddd18a885390da0a585eff1bec01677a3be

(the text expects the "wrong" result on Org 9.1.14 or later and the "correct" result on earlier versions)
